buildSrc/src/main/groovy/Classpaths.groovy

```diff
@@ -52,7 +52,7 @@ class Classpaths {
 
     static final String FLATBUFFER_GROUP = 'com.google.flatbuffers'
     static final String FLATBUFFER_NAME = 'flatbuffers-java'
-    static final String FLATBUFFER_VERSION = '2.0.3'
+    static final String FLATBUFFER_VERSION = '1.12.0'
```

Comment:
This is actually only bringing the dependency in line with our transitive dependencies - before this, we saw that _some_ classpaths had 1.12.0, and others had 1.12.0 -> 2.0.3, though none had 2.0.3 directly (except the two projects in dhc that actually used this property).

Rather than a comment, I'd like to get a little further into correct gradle dep management, and possibly a specific rule that says "if anything tries to disagree with this, blow up right away (so we correct it on a case-by-case basis".